lib/blpapi-wrapper.ts

```diff
@@ -272,15 +286,19 @@ export class Session extends events.EventEmitter {
         }).nodeify(cb);
     }
 
-    request(uri: string, name: string, request: any, callback: IRequestCallback): void {
+    request(uri: string, requestName: string, request: any, callback: IRequestCallback): void {
         this.validateSession();
 
+        var responseEventName = REQUEST_TO_RESPONSE_MAP[requestName];
+        if (responseEventName === undefined) {
+            callback(new Error('Request type ' + requestName + ' not handled'));
```

Comment:
I would re-word the error message to indicate that it is a user error, not an internal error if the request name is incorrect.  Maybe something along the lines of `Unknown request type: ' + requestName`